WordPress/gutenberg
# Modal headings and labels missing editor text color

#50448



## What?
Added Default Color of Model heading & Labels 

## Why?
Default theme color and editor text color are different so need to apply common colour for all model 

## How?
Apply default color class to model edior

## Testing Instructions
- Open any Model 
- Check default color of label and heading
